### PR TITLE
Console

### PR DESCRIPTION
**Pull request by besarism**
make pumaLogoAscii smaller
puma_console add
summarizer cleanup

### CODE REVIEW

**Comment by skyylex** [comment]
Just thinking out loud about console art, ideally the best result would be if we know the resolution of the terminal window and could scale the logo accordingly. Not sure though that it worths it. Maybe approximated size is quite a good start.

https://stackoverflow.com/a/10081642

**Comment by besarism** [comment]
> Just thinking out loud about console art, ideally the best result would be if we know the resolution of the terminal window and could scale the logo accordingly. Not sure though that it worths it. Maybe approximated size is quite a good start.

@skyylex for that then we'll need to generate ascii by ourselves and also we'll need to generate per run, what I did now is I just generated one considering the default size of a mac os terminal 80 colums x 24 rows.
But, thanks for the suggestion, let's think of something for the future releases, but for now, I think this should be enough.

File: Documentation/Develop.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ swift package generate-xcodeproj
 Open `Example/TestPuma.xcworkspace`, the Puma project has already been included. Select TestPuma scheme, in order to run
 
 - You need to sign TestPuma target, the easiest way is to check Automatic Signing and select a team
-- You need to sign all frameworks in Puma project. 
+- You need to sign all frameworks in Puma project.
```
Comment (onmyway133) [inline Documentation/Develop.md]: Now you don't need  to,  https://onmyway133.github.io/blog/How-to-fix-library-not-found-with-SPM-and-CocoaPods-in-Xcode/

Comment (besarism) [reply Documentation/Develop.md]: yeah, I will mention the solution in one of the upcoming commits

**Pull request merged**